.gitattributes

```diff
@@ -77,3 +77,6 @@ Python/stdlib_module_names.h                 generated
 Tools/peg_generator/pegen/grammar_parser.py  generated
 aclocal.m4                                   generated
 configure                                    generated
+Doc/data/stable_abi.dat                      generated
+Lib/test/test_stable_abi_ctypes.py           generated
+PC/python3dll.c                              generated
```

Comment:
Nit: can we keep this list sorted? 🤓

Reply:
Oh! I guess I saw the `a` and `c` at the end and haven't realized it's sorted. Silly me :)